package.json

```diff
@@ -19,21 +19,15 @@
     "registry": "https://registry.npmjs.org"
   },
   "scripts": {
-    "install:all": "npm ci && cd react-app && yarn --pure-lockfile",
+    "install:all": "npm ci; cd react-app; npm install; cd -",
```

Comment:
Does this `npm install` work for a react app? I remember trying like a year ago and react was not happy.